apps/antalmanac/src/components/RightPane/CoursePane/SearchForm/CourseNumberSearchBar.tsx

```diff
@@ -1,10 +1,18 @@
-import { FormControlLabel, Switch, TextField } from '@material-ui/core';
+import { FormControl, InputLabel, ListItemText, MenuItem, Select, TextField } from '@material-ui/core';
 import { ChangeEvent, PureComponent } from 'react';
 
 import RightPaneStore from '../../RightPaneStore';
 
+const courseLevelList: { level: string; label: string }[] = [
+    { level: '0', label: 'Any Course Division' },
```

Comment:
I think it makes more sense for the level here to be the same as the default level, and the drop-down should have this option selected by default.